### PR TITLE
Implement ansible modules

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: plugins/modules/pulp_ansible_repository.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = r'''
+---
+module: pulp_ansible_repository
+short_description: Manage ansible repositories of a pulp api server instance
+description:
+  - "This performes CRUD operations on ansible repositories in a pulp api server instance."
```
Comment (Timoses) [inline plugins/modules/pulp_ansible_repository.py:22]: typo: `performs`

File: plugins/modules/pulp_ansible_remote.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = r'''
+---
+module: pulp_ansible_remote
+short_description: Manage ansible remotes of a pulp api server instance
+description:
+  - "This performes CRUD operations on ansible remotes in a pulp api server instance."
```
Comment (Timoses) [inline plugins/modules/pulp_ansible_remote.py:22]: Again `performs` (also in the others except sync module)

File: plugins/modules/pulp_ansible_distribution.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            ('state', 'present', ['name', 'base_path']),
+            ('state', 'absent', ['name']),
+        ],
+    ) as module:
+
+        repository_name = module.params['repository']
+        version = module.params['version']
+
+        if module.params['content_guard']:
+            module.fail_json(msg="Content guard features are not yet supportet in this module.")
```
Comment (Timoses) [inline plugins/modules/pulp_ansible_distribution.py:122]: typo: `supported`

Comment (Timoses) [inline plugins/modules/pulp_ansible_distribution.py:122]: Should this perhaps be stated in `DOCUMENTATION` under `content_guard`?

**Comment by Timoses** [comment]
👍

File: plugins/modules/pulp_ansible_distribution.py (hunk 3 above)
Comment (mdellweg) [reply plugins/modules/pulp_ansible_distribution.py:122]: good point

**Pull request merged**